beacon-chain/sync/initial-sync-old/round_robin.go

```diff
@@ -193,7 +203,7 @@ func (s *Service) roundRobinSync(genesis time.Time) error {
 			1,                    // step
 			blockBatchSize,       // count
 			peers,                // peers
-			0,                    // remainder
+			0,                    // reminder
```

Comment:
Reminder? What does that mean in this context?

Reply:
Sorry it should have been remainder